TripHouse/main.js

```diff
@@ -47,9 +80,10 @@ const filterChildrenPlus = document.querySelector('.filter_children > .filter_pl
 const filterChildrenNumber = document.querySelector('.filter_children > .filter_plus-minus > .filter_number')
 let childrenNumber = 0;
 filterChildrenPlus.addEventListener('click', function() {
+  toggleChilndrenFilters();
   childrenNumber++;
-  if (childrenNumber >= 17) {
-    childrenNumber = 17
+  if (childrenNumber >= 10) {
+    childrenNumber = 10
```

Comment:
values like this should be used like variables - imagine that you use 10 value in 20 different places and one time it will need to replace with other value - it will be easier to change variable at one place, isn't it?)